rmgpy/chemkin.pyx

```diff
@@ -449,6 +452,25 @@ def _read_kinetics_line(line, reaction, species_dict, Eunits, kunits, klow_units
         # Duplicate reaction
         reaction.duplicate = True
 
+    elif 'COV' in line:
+        k = kinetics['arrhenius high']
+        kinetics['surface arrhenius'] = _kinetics.SurfaceArrhenius(
+            A=(k.A.value, kunits),
+            n=k.n,
+            Ea=k.Ea,
+            T0=k.T0,
+        )
+        del kinetics['arrhenius high']
+
+        tokens = tokens[1].split()
+        kinetics['coverage dependence'][species_dict[tokens[0].strip()]] = {'E': (tokens[1].strip(), Eunits), 'm': tokens[2].strip(), 'a':tokens[3].strip()}
+        try:
+            # is a sticking coefficient
+            kinetics['sticking coefficient'][species_dict[tokens[0].strip()]] = {'E': (tokens[1].strip(), Eunits), 'm': tokens[2].strip(), 'a':tokens[3].strip()}
```

Comment:
I don't see what this can be trying to do? `kinetics['sticking coefficient']` is surely not a dictionary, but if it exists it is a `StickingCoefficient` object.   Did you miss a `.coverage_dependence` ?  The try/except block is not a terribly safe way to do this, and it's presumably not tested.

Reply:
it has to do with the weird order of things when reading the file. when you read it in, COV is the last thing you will get, so it is either a surface arrhenius or sticking coefficient. the 'coverage dependence' is the surface arrhenius

Reply:
If you can find a better way to do this, I am all ears.

Reply:
This now fixed.